auto_install.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/*
+ * Please run composer install and configure htconfig.php
+ */
+
+if (PHP_SAPI !== 'cli') {
+	die("ERROR: Code execution only possible from CLI\n");
+}
+
+if (file_exists('.htconfig.php') && filesize('.htconfig.php')) {
+	die("ERROR: Installation was already executed, .htconfig.php already exists\n");
+}
+
+// Remove die from config file
+copy_config_file();
```

Comment:
The name of this function is confusing with the part where you actually copy the config file in line 105. You also are using it only once, which makes me suggest to just replace it with its actual code.